diff --git a/Makefile b/Makefile
index 3e9eb12..ad830b4 100644
--- a/Makefile
+++ b/Makefile
@@ -7,8 +7,8 @@ PREFIX = /usr/local
 BINDIR = ${PREFIX}/bin
 MANDIR = ${PREFIX}/man/man1
 
-HURL_CFLAGS = ${CFLAGS}
-HURL_LDFLAGS = -ltls ${LDFLAGS}
+HURL_CFLAGS = ${CFLAGS} -D_DEFAULT_SOURCE
+HURL_LDFLAGS = -ltls ${LDFLAGS} -lbsd
 
 SRC = hurl.c
 OBJ = ${SRC:.c=.o}
diff --git a/hurl.c b/hurl.c
index 4e0f9ab..7cbb4a6 100644
--- a/hurl.c
+++ b/hurl.c
@@ -1,4 +1,5 @@
 #include <sys/socket.h>
+#include <sys/types.h>
 #include <sys/time.h>
 
 #include <ctype.h>
@@ -10,7 +11,7 @@
 #include <stdio.h>
 #include <stdint.h>
 #include <stdlib.h>
-#include <string.h>
+#include <bsd/string.h>
 #include <time.h>
 #include <unistd.h>
 
