diff --git a/Makefile b/Makefile
index c901b0f..869d94f 100644
--- a/Makefile
+++ b/Makefile
@@ -1,11 +1,11 @@
-PREFIX=	/usr/local
+PREFIX ?= /usr/local
 
 install:
-	install -m755 bin/dir2point $(PREFIX)/bin/dir2point
-	install -m755 bin/md2point  $(PREFIX)/bin/md2point
-	install -m755 bin/point2pdf $(PREFIX)/bin/point2pdf
+	install -m755 bin/dir2point $(DESTDIR)$(PREFIX)/bin/dir2point
+	install -m755 bin/md2point  $(DESTDIR)$(PREFIX)/bin/md2point
+	install -m755 bin/point2pdf $(DESTDIR)$(PREFIX)/bin/point2pdf
 
 uninstall:
-	rm -f $(PREFIX)/bin/dir2point
-	rm -f $(PREFIX)/bin/md2point
-	rm -f $(PREFIX)/bin/point2pdf
+	rm -f $(DESTDIR)$(PREFIX)/bin/dir2point
+	rm -f $(DESTDIR)$(PREFIX)/bin/md2point
+	rm -f $(DESTDIR)$(PREFIX)/bin/point2pdf
