diff --git a/src/lib/syscall.c b/src/lib/syscall.c
index 9495a64..95525e2 100644
--- a/src/lib/syscall.c
+++ b/src/lib/syscall.c
@@ -15,13 +15,14 @@
  * Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */

+#include "torsocks.h"
+
 #include <assert.h>
 #include <stdarg.h>
 #include <sys/mman.h>

 #include <common/log.h>

-#include "torsocks.h"

 /* syscall(2) */
 TSOCKS_LIBC_DECL(syscall, LIBC_SYSCALL_RET_TYPE, LIBC_SYSCALL_SIG)
diff --git a/src/lib/torsocks.h b/src/lib/torsocks.h
index bcaf92b..490fce4 100644
--- a/src/lib/torsocks.h
+++ b/src/lib/torsocks.h
@@ -20,6 +20,10 @@
 #ifndef TORSOCKS_H
 #define TORSOCKS_H

+#if (defined(__linux__))
+#define _GNU_SOURCE 1
+#endif
+
 #include <common/compat.h>
 #include <common/config-file.h>

@@ -233,7 +237,7 @@ struct hostent **result, int *h_errnop

 #if (defined(__linux__))

-#define _GNU_SOURCE
+/* #define _GNU_SOURCE */

 /* syscall(2) */
 #define LIBC_SYSCALL_NAME syscall
