diff --git a/rar/rar.c b/rar/rar.c
index 40cafcf..edbfeaa 100644
--- a/rar/rar.c
+++ b/rar/rar.c
@@ -62,7 +62,7 @@ static bool rar_parse_entry(ar_archive *ar, off64_t offset)
                 warn("Encrypted entries will fail to uncompress");
             if ((header.flags & LHD_DIRECTORY) == LHD_DIRECTORY) {
                 if (header.datasize == 0) {
-                    log("Skipping directory entry \"%s\"", rar_get_name(ar));
+                    /* log("Skipping directory entry \"%s\"", rar_get_name(ar)); */
                     break;
                 }
                 warn("Can't skip directory entries containing data");
