From a315a269713bd18e10f0ad22b38867d63c85712c Mon Sep 17 00:00:00 2001
From: Samuel Holland <samuel@sholland.org>
Date: Thu, 18 Oct 2018 09:33:14 +0200
Subject: [PATCH 04/11] Prefer libgcc_eh over libunwind for musl

---
 src/libunwind/lib.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libunwind/lib.rs b/src/libunwind/lib.rs
index 2b3c19c06..8233beada 100644
--- a/src/libunwind/lib.rs
+++ b/src/libunwind/lib.rs
@@ -35,6 +35,6 @@ cfg_if! {
 }
 
 #[cfg(target_env = "musl")]
-#[link(name = "unwind", kind = "static", cfg(target_feature = "crt-static"))]
+#[link(name = "gcc_eh", cfg(target_feature = "crt-static"))]
 #[link(name = "gcc_s", cfg(not(target_feature = "crt-static")))]
 extern {}
-- 
2.19.1

