From 70fd5118619bf40d123162358ad0c556a99b0cdd Mon Sep 17 00:00:00 2001
From: Samuel Holland <samuel@sholland.org>
Date: Thu, 18 Oct 2018 09:37:33 +0200
Subject: [PATCH 08/11] Don't pass CFLAGS to the C++ compiler

---
 src/bootstrap/builder.rs | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/src/bootstrap/builder.rs b/src/bootstrap/builder.rs
index dc0b0aaf0..257d36975 100644
--- a/src/bootstrap/builder.rs
+++ b/src/bootstrap/builder.rs
@@ -966,7 +966,7 @@ impl<'a> Builder<'a> {
             let cflags = self.cflags(target).join(" ");
             cargo
                 .env(format!("CFLAGS_{}", target), cflags.clone())
-                .env("CFLAGS", cflags.clone());
+                .env("CFLAGS", cflags);
 
             if let Some(ar) = self.ar(target) {
                 let ranlib = format!("{} s", ar.display());
@@ -981,9 +981,7 @@ impl<'a> Builder<'a> {
                 let cxx = ccacheify(&cxx);
                 cargo
                     .env(format!("CXX_{}", target), &cxx)
-                    .env("CXX", &cxx)
-                    .env(format!("CXXFLAGS_{}", target), cflags.clone())
-                    .env("CXXFLAGS", cflags);
+                    .env("CXX", &cxx);
             }
         }
 
-- 
2.19.1

